stub/src/main/java/io/grpc/stub/ServerCalls.java

```diff
@@ -126,8 +129,12 @@ public ServerCall.Listener<ReqT> startCall(ServerCall<ReqT, RespT> call, Metadat
       Preconditions.checkArgument(
           call.getMethodDescriptor().getType().clientSendsOneMessage(),
           "asyncUnaryRequestCall is only for clientSendsOneMessage methods");
+      String preRpcAttempts = null;
```

Comment:
`previousRpcAttempts` (or at least `prevRpcAttempts`) as a name is much better than `preRpcAttempts` since pre as a short form of previous is not common